synapseAnnotations/data/ngs.json

```diff
@@ -17,6 +17,24 @@
       }
     ]
   },
+  {
+      "name": "readPair",
+      "description": "The read of origin",
+      "columnType": "STRING",
+      "maximumSize": 5,
```

Comment:
if it's a string (which maybe it doesn't need to be) then the maximum size should be 1?

Reply:
Because I wanted to be flexible for other inputs, like 'unknown'? I dunno...

Reply:
or 'R1' as kara said, but i'm happy to stick to integer.